docs/tutorials/alerting_based_on_metrics.md

```diff
@@ -6,13 +6,13 @@ sort_rank: 5
 In this tutorial we will create alerts on the `ping_request_count` metric that we instrumented earlier in the
 [Instrumenting HTTP server written in Go](./instrumenting_http_server_in_go.md) tutorial.
 
-For the sake of this tutorial we will alert when the `ping_request_count` metric is greater than 5, Checkout real world [best practices](../practices/alerting.md) to learn more about alerting principles.
+For the sake of this tutorial we will alert when the `ping_request_count` metric is greater than 5. Check out real world [best practices](../practices/alerting.md) to learn more about alerting principles.
 
-Download the latest release of Alertmanager for your operating system from [here](https://github.com/prometheus/alertmanager/releases)
+Download the latest release of Alertmanager for your operating system from [here](https://github.com/prometheus/alertmanager/releases).
 
-Alertmanager supports various receivers like `email`, `webhook`, `pagerduty`, `slack` etc through which it can notify when an alert is firing. You can find the list of receivers and how to configure them [here](/docs/alerting/latest/configuration/). We will use `webhook` as a receiver for this tutorial, head over to [webhook.site](https://webhook.site) and copy the webhook URL which we will use later to configure the Alertmanager.
+Alertmanager supports various receivers like `email`, `webhook`, `pagerduty`, `slack` etc through which it can notify when an alert is firing. You can find the list of receivers and how to configure them [here](../../alerting/latest/configuration). We will use `webhook` as a receiver for this tutorial, head over to [webhook.site](https://webhook.site) and copy the webhook URL which we will use later to configure the Alertmanager.
```

Comment:
The `/docs/alerting/...` subtree is pulled from another repo (Alertmanager), so for that it would be best to keep the link as it was. I think I'll explicitly mention in the Markdown guidelines which paths are external, so everyone is aware of what comes from where :)

Reply:
Addressed in [7be94bc](https://github.com/prometheus/docs/pull/2667/commits/7be94bc0024fd0b09767ee2077f487093ad646de)